tsconfig.json

```diff
@@ -3,9 +3,10 @@
     "target": "ES2018",
     "module": "commonjs",
     "lib": ["ES2018"],
+    "esModuleInterop": true,
     "declaration": true,
     "sourceMap": true,
-    "removeComments": true,
+    "removeComments": false,
```

Comment:
Why the change?

Reply:
`jsdoc2md` can't parse the `.ts` files, so it's configured to parse the generated `.js` files instead. When the comments were being stripped out, though, `jsdoc2md` didn't have anything to parse.

The `jsdoc2md` docs say that you _may_ be able to use `jsdoc2md` and babel to use babel to compile your `.ts` files and have `jsdoc2md` build docs off of those, but there are a number of documented cases where that doesn't work. I ran into enough of those cases that I decided to just build the docs off of the `.js` files that `tsc` compiled.